screener.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  projectRepo: 'Esri/calcite-components',
+  storybookConfigDir: '.storybook',
+  storybookStaticDir: './__docs-temp__',
+  apiKey: process.env.SCREENER_API_KEY,
+  resolution: '1024x768',
+  browsers: [
+    {
+      browserName: 'chrome'
```

Comment:
Are we only testing with chrome at one resolution to start?

Reply:
I think so, just to see how many screenshots we end up generating for this initial adoption.  Not sure if the resolution should be tweaked or not.  Might make sense to make the resolution smaller for responsiveness since it it used in a lot of tight places.  These are basic defaults that I didn't change but as we see how this goes it makes a lot of sense to tweak these to our needs.